{{cookiecutter.project_name}}/server/settings/components/common.py

```diff
@@ -13,6 +13,7 @@
 from typing import Dict, List, Tuple, Union
 
 from server.settings.components import BASE_DIR, config
+from django.utils.translation import ugettext_lazy as _
```

Comment:
Missed newline between import groups.

Reply:
I did everything right? 🙂

Reply:
Not exactly.
I added the correct code of importing ;)
Could you please change.